compiler/build.gradle

```diff
@@ -88,7 +88,7 @@ publishing {
             artifacts = []
             artifactId 'protoc-gen-grpc-kotlin'
             artifact("$buildDir/artifacts/protoc-gen-grpc-kotlin" as File) {
-                classifier "jdk8"
+                classifier "jdk7"
```

Comment:
👍 In addition to fixing the build, this also matches the min SDK requirement stated in the docs: https://grpc.io/docs/languages/kotlin/quickstart/#prerequisites